ci/verify-chef.sh

```diff
@@ -87,7 +87,7 @@ if [ ! -f "Gemfile.lock" ]; then
 fi
 
 unset CHEF_FIPS
-if [ $PIPELINE_NAME="chef-fips" ]; then
+if [ "$PIPELINE_NAME" = "chef-fips" ]; then
```

Comment:
i usually use `if test "$whatever"  = "stuff"; then` for maximum old-shell paranoia